rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py

```diff
@@ -2,6 +2,23 @@
 import numpy as np
 
 
+def cleanup_tokens(
+    token_ids_string: List[Tuple[int, Text]], delimiter: Text
+) -> Tuple[List[int], List[Text]]:
+    """Utility method to apply specific delimiter based cleanup on list of tokens"""
+
+    token_ids_string = [
+        (id, string.replace(delimiter, "")) for id, string in token_ids_string
+    ]
+
+    # remove empty strings
+    token_ids_string = [(id, string) for id, string in token_ids_string if string]
+
+    # return as individual token ids and token strings
+    token_ids, token_strings = zip(*token_ids_string)
+    return token_ids, token_strings
```

Comment:
```suggestion
    return zip(*token_ids_string)
```

Reply:
@tabergma Get a type error if I do that, that's why had to first unpack it and then return.

Reply:
ok, thanks for the explanation, then I guess it is fine